frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -87,7 +87,7 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
 
   const afterStart = (name: string, type: 'created' | 'updated') => {
     const { acceleratorProfile, notebookSize, image } = startNotebookData;
-    fireTrackingEventRaw(`Workbench ${type === 'created' ? 'Created' : 'Updated'}`, {
+    const tep: FormTrackingEventProperties = {
```

Comment:
There are several created items here and I think we need to figure out a way to just pass it to the event call. If we need to build this complexity, I wonder if we should look at making custom hooks and utilities to craft this to avoid it polluting the forms with this functionality. 

We may need to look at all the events after we have done the effort to make sure we aren't losing some good refactoring and cleanup items. The goal is to structure all these in such a fashion to support segment more cleanly throughout our app -- so maybe we need to have higher layer functions for each form or something. Not sure. Like I said, we need to work through this once we have an understanding of each usage better.

Reply:
I am not sure if this is now an ask to just inline `tep` or a forward looking comment for future work (potentially outside this PR).

Reply:
The variable was a bit odd, but basically this kind of data probably needs to be filtered through a function and not so "on the form" so segment functionality is a little less "in the middle of stuff" and more documented for the use-cases it wants to provide. Definitely future work.